pkgs/development/python-modules/notifications-android-tv/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "notifications-android-tv";
+  version = "1.0.0";
+  format = "pyproject";
+
```

Upstream only supports [Python 3.8](https://github.com/engrbm87/notifications_android_tv/blob/dev/pyproject.toml#L21). I would like to suggest to follow them.

```suggestion
  format = "pyproject";

  disabled = pythonOlder "3.8";
```